### PR TITLE
Fix a minor formatting bug in the atm-correction RTD pages

### PR DESCRIPTION
**Pull request by adybbroe**
This PR is supposed to fix a very minor formatting bug in the RTD pages. It concerns only one unite "micrometers" on the atm-correction pages.

 - [ ] Closes #xxxx 
 - [ ] Tests added 
 - [ ] Tests passed: Passes ``pytest pyspectral`` 
 - [ ] Passes ``flake8 pyspectral`` 
 - [ ] Fully documented

### CODE REVIEW

**Review by mraspaud — APPROVED** [review]
LGTM

**Comment by mraspaud** [comment]
Looks like there is a dependency problem in the unstable build...

**Comment by djhoese** [comment]
Looks like GDAL 3.1.0+ isn't available through the normal apt installation that you're currently using. You could try installing it from conda then installing rasterio from github. It looks like conda-forge has as new as GDAL 3.5 available.

**Comment by adybbroe** [comment]
> Looks like GDAL 3.1.0+ isn't available through the normal apt installation that you're currently using. You could try installing it from conda then installing rasterio from github. It looks like conda-forge has as new as GDAL 3.5 available.

Ok, thanks! I am just trying to understand why I should need GDAL and rasterio at all for Pyspectral.

**Comment by djhoese** [comment]
Good point. You probably don't. The unstable/experimental environment setup was probably copied from Satpy or pyresample. If you don't have rasterio in your regular environments then I'd say you can remove the `gdal-dev` and `rasterio` installs in the unstable environment creation.

**Comment by simonrp84** [comment]
Changes look good to me, but I did notice one typo in a different line of the docs!

**Comment by mraspaud** [comment]
@simonrp84 which line? you don't seem to have commented in the code, right?

**Comment by simonrp84** [comment]
The inline review comment just above my previous comment.

File: doc/rayleigh_correction.rst [hunk 1]
```diff
@@ -81,7 +81,7 @@ provide a more gentle scaling in cases of high reflectances (above 20%):
 In case you want to bypass the reading of the sensor response functions or you have
 a sensor for which there are no RSR data available in PySpectral it is still possible
 to derive an atmospheric correction for that band. All what is needed is the effective
-wavelength of the band, given in micrometers (`:math: \mu m`). This wavelength is
+wavelength of the band, given in micrometers (:math:`\mu m`). This wavelength is
 normally calculated by PySPectral from the RSR data when passing the name of the band
```
Comment (adybbroe) [inline doc/rayleigh_correction.rst:85]:
```suggestion
normally calculated by PySpectral from the RSR data when passing the name of the band
```

**Comment by adybbroe** [comment]
> The inline review comment just above my previous comment.
? Don't see anything here I am sorry @simonrp84 Please help 
I found one myself though, perhaps the same?

**Pull request merged**